tests/11register.pl

```diff
@@ -698,3 +749,80 @@ sub add_email_for_user {
       });
    };
 
+test "Can register using an email address via identity server",
+   requires => [ $main::API_CLIENTS[0], localpart_fixture(), id_server_fixture() ],
+
+   do => sub {
+      my ( $http, $localpart, $id_server ) = @_;
+
+      my $email_address = 'testemail@example.com';
+
+
+      # print "\n\n$BIND_HOST\n\n\n";
+      # print "\n\n${id_server->name}\n\n\n";
+      $http->do_request_json(
+         method => "POST",
+         uri    => "/r0/register",
+
+         content => {
+            username => $localpart,
+            password => "noobers3kr1t",
+            device_id => "xyzzy",
+         },
+      )->main::expect_http_401->then( sub {
+         my ( $response ) = @_;
+
+         my $body = decode_json $response->content;
+
+         assert_json_keys( $body, qw( session flows ));
+
+         log_if_fail "No single m.login.email.identity stage registration flow found", $body;
```

Comment:
```suggestion
         log_if_fail "First /register body", $body;
```

Since we may fail for other reasons.